src/js/tabs/pid_tuning.js

```diff
@@ -2100,6 +2100,12 @@ TABS.pid_tuning.updateRatesLabels = function() {
                 {value: parseInt(maxAngularVelPitch), balloon: function() {drawBalloonLabel(stickContext, maxAngularVelPitch, curveWidth, rateScale * (maxAngularVel - parseInt(maxAngularVelPitch)), 'right', BALLOON_COLORS.pitch, balloonsDirty);}},
                 {value: parseInt(maxAngularVelYaw), balloon: function() {drawBalloonLabel(stickContext, maxAngularVelYaw,   curveWidth, rateScale * (maxAngularVel - parseInt(maxAngularVelYaw)),   'right', BALLOON_COLORS.yaw, balloonsDirty);}}
             ];
+            // show warning message if angular velocity exceeds
+            if (parseInt(maxAngularVelRoll) > 1800 || parseInt(maxAngularVelPitch) > 1800 || parseInt(maxAngularVelYaw) > 1800) {
+                $('.maxRateWarning').show();
+            } else {
+                $('.maxRateWarning').hide();
+            }
```

Comment:
To make it cleaner, is better to:
- Define a variable type `const` with the value, in this way it can be changed in only one place. Something like `const MAX_RATE_WARNING = 1800;`
- Assign all the conditional to one variable.
- Use a `.toggle(variable)` in place of all `if-else`

Reply:
yeah, they update code all the time

Look at what they are doing with Kotlin and C# for AR.  Constantly depreciating obsolete code.  If you use a new code on an older version, it won't work.  If you use older code on a new version w/ depreciated code, it won't work.  As long as it works and works every time, it doesn't matter.  The cleaner the better though.